src/vehicles/RosVehicle.lua

```diff
@@ -56,9 +56,18 @@ function RosVehicle:onLoad()
     local spec = self.spec_rosVehicle
     -- rosVehicle variables
     -- .id is initialized by FS
-    spec.ros_veh_name = ros.Names.sanatize(self:getFullName() .. "_" .. self.id)
-    spec.base_link_frame = spec.ros_veh_name .. "/base_link"
-
+    spec.ros_veh_name_w_id = ros.Names.sanatize(self:getFullName() .. "_" .. self.id)
+    spec.ros_veh_name = ros.Names.sanatize(self:getFullName())
+    spec.base_link_frame = spec.ros_veh_name_w_id .. "/base_link"
+    spec.l_v_x_0 = 0
+    spec.l_v_y_0 = 0
+    spec.l_v_z_0 = 0
+    spec.sec = 0
```

Comment:
What's this all for?

Reply:
Initialization of linear velocity and time(s) for Imu messages. I've added a comment explaining what it is for.